src/index.js

```diff
@@ -287,6 +287,27 @@ const replaceInWindows = (
           )
         : textBetweenDelimiters
 
+      const isItalics = delimiterLiteral === '_' && options.spacePadded
+      if (isItalics) {
+        /**
+         * If there is an isolated underscore, then we don't want to replace
+         * it with empty <em> tags and keep them as is
+         */
+        if (!/[^\s_]/.test(textBetweenDelimiters)) {
+          // Skip this match, move to next
+          return replaceInWindows(
+            text,
+            delimiterLiteral,
+            replacementOpeningLiteral,
+            replacementClosingLiteral,
+            closedTagWindows,
+            options,
+            tagWindowIndex,
+            tagWindowOffset + openingMatch[0].length
+          )
+        }
+      }
```

Comment:
What if we replace this with the following?
Tested that all tests are passing. And this makes sense to have for all the formatting that slack supports
```suggestion
       if (textBetweenDelimiters.length === 0) {
        return replaceInWindows(
          text,
          delimiterLiteral,
          replacementOpeningLiteral,
          replacementClosingLiteral,
          closedTagWindows,
          options,
          tagWindowIndex,
          tagWindowOffset + openingMatch[0].length
        );
      }
```

Also we can move this if check at L281

Reply:
makes sense
updated